test/reconstruction/community.jl

```diff
@@ -24,12 +24,7 @@
     @test all(lb[1:14] .== -ub[1:14] .== -1000)
     @test all(lb[15:16] .== -ub[15:16] .== 0.0)
 
-    c2 = join_with_exchanges(
-        [m1, m2],
-        ex_rxn_mets;
-        add_biomass_objective = true,
-        biomass_ids = ["biomass1", "biomass1"],
-    )
+    c2 = join_with_exchanges([m1, m2], ex_rxn_mets; biomass_ids = ["biomass1", "biomass1"])
```

Comment:
is this right? (2x the same "biomass")

Reply:
yes, both models all their biomass function "biomass1"

Reply:
ah so cool :D